vacasa/connect/util.py

```diff
@@ -10,15 +10,24 @@
 
 def log_http_error(r: Response):
     """ Log any 4XX/5XX error responses and re-raise """
-    try:
-        r.raise_for_status()
-    except RequestException as e:
+    if r.status_code < 500:
         try:
-            logger.exception(r.json())
-        except ValueError:
-            logger.exception(r.content)
-        raise e
-
+            r.raise_for_status()
+        except RequestException as e:
+            try:
+                logger.info(r.json())
+            except ValueError:
+                logger.info(r.content)
+            raise e
+    else:
+        try:
+            r.raise_for_status()
+        except RequestException as e:
+            try:
+                logger.exception(r.json())
```

Comment:
Not sure why we were doing this rather than just passing e to logger.exception, but kept it the same.